spec/plans/context.fmf

```diff
@@ -0,0 +1,33 @@
+summary: Definition of the test execution context
+
+story:
+    As a user I want to define the default context in which tests
+    included in given plan should be executed.
+
+description: |
+    Define the default context values for all tests executed under
+    the given plan. This overrides context values auto-detected
+    from the environment, such as ``distro``, ``variant`` or
+    ``arch`` but can overriden by context provided directly on the
+    command line. See the :ref:`/spec/context` definition for the
+    full list of supported context dimensions.
+
+example: |
+    summary:
+        Test basic functionality of the httpd24 collection
+    discover:
+        how: fmf
+    execute:
+        how: tmt
+    context:
+        collection: httpd24
+
+    summary:
+        Verify dash against the shared shell tests repository
+    discover:
+        how: fmf
+        url: https://src.fedoraproject.org/tests/shell
+    execute:
+        how: tmt
+    context:
+        component: dash
```

Comment:
What effect this could have. Do you think this is to be used in `prepare` for the package / component installation? Can you give me an example of use for this?

Reply:
By this you can set context as you need in your plan (L2) metadata and make adjustments in the discover accordingly. IMHO good for your own dimensions.

Reply:
So to extend the example: in plan you set context: component = dash
And in test you can have:
```
adjust:
  - when: component == dash
    require+: [dash]
    environment+:
      SH: dash
  - when: component == ksh
    require+: [ksh]
    environment+:
      SH: ksh
```